src/main/scala/com/microsoft/ml/spark/lightgbm/LightGBMBooster.scala

```diff
@@ -46,8 +46,22 @@ class LightGBMBooster(val model: String) extends Serializable {
     }
   }
 
+  def featuresShap(features: Vector): Array[Double] = {
```

Comment:
I think I will need to add a pyspark API method to expose this method to the pyspark python wrapper (I don't think the pyspark methods are autogenerated from the Models directly, only params are for Estimators)

Reply:
doesn't need to be done in this PR, I can add it later.  Do you use the scala API primarily?

Reply:
If you find this PR and feature useful, yes we can do this. It was just  not in my mind when I came with this PR

Reply:
I do not use the Python API regularly

Reply:
Another question is if thia feature should also be made available to LightGBMRegressor and LightGBMClassifier

Reply:
Another question is if thia feature should also be made available to LightGBMRegressor and LightGBMClassifier?

Reply:
yes, it definitely should eventually be added to all of them. adding it to ranker in this PR is a good start.